packages/background/src/controllers/BlankController.ts

```diff
@@ -1355,6 +1359,33 @@ export default class BlankController extends EventEmitter {
         return true;
     }
 
+    /**
+     * Resets an account by removing its transaction history and added tokens.
+     *
+     * @param address address to be reset - hex
+     */
+    private async accountReset({
+        address,
+    }: RequestAccountRemove): Promise<void> {
+        // Reset account
+        await Promise.all([
+            this.transactionController.wipeTransactionsByAddress(address),
```

Comment:
`wipe` - `reset` - `remove`.

Can we standardize the verbs? I would use: 

- `Reset` for transactions and tokens. 
- `Revoke` for permissions

Also, we shouldn't we also cleanup the `BridgeController` transactions?

Reply:
On it sir, Great catch!

Reply:
Done 💪